scripts/update/requestrr.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+. /etc/swizzin/sources/functions/os
+if [[ -f /install/.requestrr.lock ]]; then
+    systemctl disable --now requestrr
+    echo_progress_start "Downloading source files"
+    case "$(_os_arch)" in
+        "amd64") dlurl=$(curl -sNL https://api.github.com/repos/darkalfx/requestrr/releases/latest | grep -Po 'ht(.*)linux-x64(.*)zip') >> ${log} 2>&1 ;;
+        "armhf") dlurl=$(curl -sNL https://api.github.com/repos/darkalfx/requestrr/releases/latest | grep -Po 'ht(.*)linux-arm(.*)zip') >> ${log} 2>&1 ;;
+        "arm64") dlurl=$(curl -sNL https://api.github.com/repos/darkalfx/requestrr/releases/latest | grep -Po 'ht(.*)linux-arm64(.*)zip') >> ${log} 2>&1 ;;
+        *)
+            echo_error "Arch not supported"
+            exit 1
+            ;;
+    esac
+
+    if ! curl "$dlurl" -L -o /tmp/requestrr.zip >> "$log" 2>&1; then
+        echo_error "Download failed, exiting"
+        exit 1
+    fi
```

Comment:
just a stylistic thing, but we're using ` || {}` for this

```suggestion
    curl "$dlurl" -L -o /tmp/requestrr.zip >> "$log" 2>&1 || {
        echo_error "Download failed, exiting"
        exit 1
    }
```
looks slicker imo